### DISCUSSION TITLE
Scalability of using MongoDB and Docker for session management

### DISCUSSION THREAD

**Discussion by Francuchin** [WhiskeySockets/Baileys#155] [General]
I am using this library in my own system, saving sessions with MongoDB on a server with Docker.
Currently, it manages around 4 or 5 sessions.
How scalable is this approach? In other words, how many connections could it handle from the same server? Has anyone had a similar experience?
Thank you for your comments.

**Comment by Fonnte** [comment]
Don't worry, it's pretty much scalable
